cvat/apps/engine/models.py

```diff
@@ -334,7 +334,9 @@ class Project(models.Model):
     target_storage = models.ForeignKey('Storage', null=True, default=None,
         blank=True, on_delete=models.SET_NULL, related_name='+')
 
-    def get_labels(self):
+    def get_labels(self, prefetch_attributes=False):
```

Comment:
This seems unnecessary. Couldn't the caller just use `x.get_labels().prefetch_related('attributespec_set')` for the same result?